src/pages/BlockDetail/BlockComp.tsx

```diff
@@ -90,7 +90,12 @@ const BlockMinerReward = ({
   )
 }
 
-export const BlockOverviewCard: FC<{ blockHeightOrHash: string; block: Block }> = ({ blockHeightOrHash, block }) => {
+export const BlockOverviewCard: FC<{
+  blockHeightOrHash: string
+  block: Block
+  blockNumber: number
```

Comment:
This property now has no difference from `block.number` and should not be retained. Otherwise, future developers may find it confusing.

Reply:
> This property now has no difference from `block.number` and should not be retained. Otherwise, future developers may find it confusing.

Not the same